app/assets/javascripts/annotations.js

```diff
@@ -1020,6 +1020,7 @@ var makeAnnotationMovable = function(annotationEl, annotationObj) {
     var curWidth = (positionArr[3] || 120);
     var curHeight = (positionArr[4] || 60);
 
+    /** 
```

Comment:
Are you sure we want to remove this function? Now it is no longer possible to drag an annotation to change its position on the document. This seems to me like a feature regression, unless I missed a discussion about removing this feature.

Reply:
I intentionally wanted to remove the ability to drag around the annotation to streamline the adding annotation process.

Reply:
When we create the pdf on a ubuntu based deployment, the pdf comment does show up, even when downloaded onto a Mac OS X. After discussion we decided that making this feature work on a mac os x deployment of autolab is not worth the time, as of now (because Autolab would most likely be deployed onto a Ubuntu system).